mar/tiebout-action.hoon

```diff
@@ -0,0 +1,40 @@
+::
+::::  /hoon/actions/collection/mar
+  ::
+/?  309
+/-  hall
+/+  tiebout
+=,  tiebout
+=,  format
+::
+|_  act=action:tiebout
+::
+++  grow
+  |%
+  ++  tank  >act<
+  --
+::
+++  grab
+  |%
+  ++  noun  action:tiebout
+  ++  json
+    |=  jon=^json
+    ^-  action:tiebout
+    %-  (hard action:tiebout)
```

Comment:
Do you *need* to hard this? If you write your json parsing correctly it should just nest.